tezos/context/src/working_tree/string_interner.rs

```diff
@@ -84,16 +125,100 @@ impl BigStrings {
     }
 
     fn clear(&mut self) {
-        let cap = self.strings.capacity();
+        // let cap = self.strings.capacity();
```

Comment:
A bunch of commented out code here and in the next methods. Should not be called anymore or needs to be reimplemented?